src/main/msp/msp.c

```diff
@@ -2815,6 +2817,12 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
 #endif
         }
         pidInitConfig(currentPidProfile);
+        // added 2021_04_25
+#if defined(USE_INTERPOLATED_SP)
+            currentPidProfile->ff_jitter_factor = sbufReadU8(src);
+#else
+            sbufReadU8(src);
+#endif
```

Comment:
Comment not needed. 
This should go inside the if statement above where all the other parameters added in 1.44 already are. 
The if statement should also be updated to reflect how many bytes are left. Since the parameter you're adding is one uint8_t the remaining bytes should be incremented from 5 to 6